### PR TITLE
Explain how to ensure effective redaction

### PR DESCRIPTION
**Pull request by robstradling**
(and how to avoid futile redaction)

### CODE REVIEW

**Comment by AGWA** [comment]
:+1:

**Comment by robstradling** [comment]
@eranmes @AGWA Would it make sense to move this to the Security Considerations section?

**Comment by AGWA** [comment]
@robstradling I don't like the implication that redaction is a security mechanism.  It could be a Privacy Consideration, although it also seems fine where it is currently.

File: rfc6962-bis.xml [hunk 1]
```diff
@@ -401,6 +401,22 @@ REDACT(label) = prefix || BASE32(index || LABELHASH(keyid || label))</artwork>
...
+              <t>
+                Avoid registering private labels in public DNS.
+              </t>
+              <t>
+                Avoid using private labels that are predictable (e.g. <spanx style="verb">www</spanx>).
+              </t>
+            </list>
+          </t>
+          <t>
+            CAs SHOULD carefully consider each request to redact a label. When a CA believes that redacting a particular label would be futile, the CA SHOULD NOT redact it. TLS clients may have policies that forbid redaction, so redaction should only be used when it's absolutely necessary and likely to be effective.
```
Comment (eranmes) [inline rfc6962-bis.xml]: s/SHOULD/should/g, as it's not a part of the protocol, but a recommendation for CAs.

**Comment by eranmes** [comment]
I agree with Rob it'd make sense to move it to another section, probably the Security Considerations section (which could be renamed Security & Privacy considerations). Since this text does not prescribe specific behaviour of a party in the protocol, it shouldn't be among the technical details of how to do redaction (it may go unnoticed there).

File: rfc6962-bis.xml (hunk 1 above)
Comment (robstradling) [inline rfc6962-bis.xml]: Ditto for "it is RECOMMENDED that domain owners", I suppose.

**Comment by robstradling** [comment]
@eranmes PTAL

**Comment by eranmes** [comment]
LGTM, please squash before merging (you can now do that with the green button).

**Comment by robstradling** [comment]
Merged at https://github.com/google/certificate-transparency-rfcs/commit/306817128875be6c7b232c7a85da79d418785935

**Pull request closed without merging**